<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aufgabe2_1</class>
 <widget class="QMainWindow" name="Aufgabe2_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aufgabe2_1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="picLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="load12bitButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 2D</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCenter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1024</number>
    </property>
    <property name="maximum">
     <number>3071</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderWidth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_center">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Center: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_width">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>580</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width: 800</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load3DButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>40</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Slice View</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderSlice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>129</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_slice">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image3D">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderThreshold">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1024</number>
    </property>
    <property name="maximum">
     <number>3071</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_threshold">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>660</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold: 300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="render3DButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>70</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Render 3D</string>
    </property>
   </widget>
   <widget class="QLabel" name="zposLabel">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>500</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yposLabel">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>470</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xposLabel">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>440</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coordinates</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMedizintechnik_Praktikum">
    <property name="title">
     <string>Medizintechnik Praktikum</string>
    </property>
   </widget>
   <addaction name="menuMedizintechnik_Praktikum"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
